.github/workflows/pkgdown.yaml

```diff
@@ -42,7 +49,7 @@ jobs:
 
       - name: Deploy to GitHub pages 🚀
         if: github.event_name != 'pull_request'
-        uses: JamesIves/github-pages-deploy-action@v4.6.0
+        uses: JamesIves/github-pages-deploy-action@v4.5.0
```

Comment:
why role this back?

Reply:
I don't know. I used `usethis::use_pkgdown_github_pages()` and it automatically made these changes. Happy to revert this back forward.

Reply:
yes please do.